kolibri/core/auth/serializers.py

```diff
@@ -92,11 +93,37 @@ class Meta:
             "allow_guest_access",
             "registered",
             "preset",
+            "num_users_in_facility",
         )
 
     def get_allow_guest_access(self, instance):
         return allow_guest_access()
 
+    def get_num_users_in_facility(self, instance):
+        facility = Facility.objects.get(dataset_id=instance.id)
+        return FacilityUser.objects.filter(facility=facility).count()
+
+
+class FacilityUsersForFacilitiesSerializer(serializers.Serializer):
```

Comment:
Can probably just do this inline in the method above, no need to define a serializer here.